tests/governance.py

```diff
@@ -689,6 +852,15 @@ def add(parser):
 
     infra.log_capture.COLORS = False
 
+    cr.add(
+        "session_coseauth",
+        gov,
+        package="samples/apps/logging/liblogging",
+        nodes=infra.e2e_args.max_nodes(cr.args, f=0),
+        initial_user_count=3,
+        authenticate_session="COSE",
+    )
```

Comment:
When this instance runs, the `authenticate_session="COSE"` arg means that _all_ of the governance auth happens with COSE signatures, right? So what are the manual `test_cose_*` functions doing differently?

Reply:
They are sent over TLS authenticated sessions, strangely. But I will refactor this completely, because it's very odd and disjoint in its current form.